src/cli/main.ts

```diff
@@ -74,24 +82,32 @@ const main = (...args: string[]) => {
   const resultNames: string[] = [];
   const resultSets: Row[][] = [];
 
-  directoryResultSets.forEach(({ name, server, client }) => {
+  directoryResultSets.forEach(({ name, server, client, interactions }) => {
     const resultName = path.basename(name);
     stdout(resultName);
 
     const serverResults = prepRows(server);
     const clientResults = prepRows(client);
 
+    const hasInteractionsResults = Object.entries(interactions).length > 0;
+    const interactionsResults = hasInteractionsResults ? prepRows(interactions) : [];
+
     stdout('Serverside');
     printCSV(serverResults);
 
     stdout('Clientside');
     printCSV(clientResults);
 
+    if (hasInteractionsResults) {
```

Comment:
yeah if the heading and printCSV logic were together this would probably be able to be dealt with cleanly in the printCSV function ay